### ISSUE TITLE
Install and configure JaCoCo plugin (Jenkins)

### ISSUE THREAD

**Issue by FRosner** [FRosner/DataGenerator#198]
It would be nice to have a report documenting our code coverage of the tests. The [JaCoCo](https://wiki.jenkins-ci.org/display/JENKINS/JaCoCo+Plugin) plugin offers such functionality.

---

Depends on: #168
